src/modules/modals/EditRow.vue

```diff
@@ -17,6 +17,10 @@
 					type="error">
 					{{ t('tables', '"{columnTitle}" should not be empty', { columnTitle: column.title }) }}
 				</NcNoteCard>
+				<NcNoteCard v-if="localRow[column.id] && column.type === 'text-link' && !isValidUrlProtocol(localRow[column.id])"
+					type="error">
+					{{ t('tables', 'Invalid protocol. Allowed: {allowed}', {allowed: allowedProtocols.join(', ')}) }}
```

Comment:
```suggestion
					{{ t('tables', 'Invalid protocol. Allowed: {allowed}', {allowed: allowedProtocols.map(p => p.replace(/:$/,'')).join(', ') }}
```

Could make the list a bit easier to read without the extra `:`